src/uwtools/tests/drivers/test_ww3.py

```diff
@@ -0,0 +1,91 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+WW3 driver tests.
```

Comment:
```suggestion
WaveWatchIII driver tests.
```
or `ww3 driver tests` -- we're using `ww3` as an in-code identifier for this component, which makes sense because `wwiii` is kind of awful (is that a Nintendo product?), but we could be consistent in our human-readable name.